dataikuapi/dss/notebook.py

```diff
@@ -27,14 +28,20 @@ def unload(self, session_id=None):
                 raise Exception("Several sessions of the notebook are running, choose one")
             else:
                 session_id = state['activeSessions'][0].get('sessionId', None)
-        return self.client._perform_json("DELETE", "/projects/%s/notebooks/" % self.project_key, params={'notebookName' : self.notebook_name, 'sessionId' : session_id})
+        return self.client._perform_json("DELETE",
+                                         "/projects/%s/jupyter-notebooks/%s/sessions/%s" % (self.project_key, self.notebook_name, session_id))
 
     def get_state(self):
         """
         Get the status of the notebook
```

Comment:
This is (and was) not compatible with SQL notebooks
```suggestion
        Get the status of the Jupyter notebook
```